packages/server/src/modules/User/UserMutations/createUser.ts

```diff
@@ -0,0 +1,59 @@
+import { GraphQLNonNull, GraphQLString } from 'graphql';
+import { mutationWithClientMutationId } from 'graphql-relay';
+import { UserModel } from '../userModel';
+import { UserType } from '../userType';
+import { genSalt, hash } from 'bcryptjs';
+import jwt from 'jsonwebtoken';
+
+export const createUser = mutationWithClientMutationId({
+  name: 'createUser',
+  inputFields: {
+    email: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'User email',
+    },
+    password: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'User password',
+    },
+    fullName: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'User full name',
+    },
+  },
+  outputFields: {
+    user: {
+      type: UserType,
+      resolve: (response) => response.user,
+    },
+    token: {
+      type: GraphQLString,
+      resolve: (response) => response.token,
+    },
+  },
+  mutateAndGetPayload: async ({ ...payload }) => {
```

Comment:
`(payload) is the same of ({ ...payload ])`

I like to call as args

Reply:
Ok,  I'll add types too since I did something similar in the login mutation 👇🏻 
```typescript
interface ILoginArgs {
  email: string;
  password: string;
}


mutateAndGetPayload: async ({ ...args }: ILoginArgs)
```